cdci_data_analysis/analysis/parameters.py

```diff
@@ -583,6 +583,18 @@ def check_name_value(value, units=None, name=None, par_format=None):
 class Name(String):
     owl_uris = String.owl_uris + ("http://odahub.io/ontology#AstrophysicalObject",)
 
+class FileReference(String):
+    owl_uris = "http://odahub.io/ontology#FileReference"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_default_value(self):
+        return {'value': self.value, 'downloadable': True}
```

Comment:
You can make this configurable for the future as a parameter (with default True) in the `__init__`.